### PR TITLE
add transaction macro to helpers

### PR DESCRIPTION
**Pull request by MarketaAdamova**

### CODE REVIEW

File: src/utility_belt/sql/helpers.clj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(ns utility-belt.sql.helpers
+  (:require [clojure.java.jdbc :as jdbc]))
+
+
+(defmacro with-db-transaction
```
Comment (MarketaAdamova) [inline src/utility_belt/sql/helpers.clj]: @lukaszkorecki I couldn't find the original macro you created. This seems to work, but it is really just a wrapper (so the only advantage is we get stable version of JDBC library).

Comment (lukaszkorecki) [reply src/utility_belt/sql/helpers.clj]: Nice! I think we can drop the "db" part from the macro name. My original implementation looked like this:

```clojure
(defmacro with-tx
  "Sugar for running queries within a transaction.
  Usage:
  (with-tx [tx db]
    (let [new (create-account! tx)]
      (create-user! tx (:id new))))"
  [[tx-name connection] & body]
  `(jdbc/with-db-transaction [~tx-name ~connection]
     ~@body))
```

I didn't think of treating the binding like a regular argument. Regardless, we need tests to make sure that when we switch to jdbc.next we have all cases covered.

Comment (MarketaAdamova) [reply src/utility_belt/sql/helpers.clj]: Ok `with-transaction` sounds good and I'll try to add some tests (I guess something like testing transaction did rollback?)

Comment (lukaszkorecki) [reply src/utility_belt/sql/helpers.clj]: Exactly 👍

**Review by lukaszkorecki — COMMENTED** [review]
Let's rename the fn + tests, and we're good to go 🚀

**Comment by MarketaAdamova** [comment]
@lukaszkorecki actually about the changelog. We use README in bunnicla but that prob not the best idea?

https://github.com/nomnom-insights/nomnom.bunnicula/blob/master/README.md

**Comment by lukaszkorecki** [comment]
@MarketaAdamova agreed, a separate changelog file is definitely preferable. [This is a nice guide on how to structure one](https://keepachangelog.com/en/1.0.0/).

I'm planning to restructure bunnicula, caliban and stature to follow the newer project format: docs in the readme (if possible), separate changelog file etc.

File: Changelog.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Changelog
```
Comment (MarketaAdamova) [inline Changelog.md:1]: @lukaszkorecki something like this

Comment (lukaszkorecki) [reply Changelog.md:1]: Perfect 👍

**Comment by MarketaAdamova** [comment]
@lukaszkorecki is this ok to merge?

**Comment by MarketaAdamova** [comment]
@lukaszkorecki  ha sorry I realised I did not push the last commit, only deployed the repo.

**Pull request merged**